### PR TITLE
Alex/update style sheets

### PR DESCRIPTION
**Pull request by alexgomezlf**
Update style sheets to be more specific
#41

### CODE REVIEW

File: projects/styles/lf-laserfiche-lite/sass/_dropdown-lf.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 @use "typography-lf";
 
-$border-color: #E0E0E0;
+$border-color: #e0e0e0;
```
Comment (Ke-Jin-Laserfiche) [inline projects/styles/lf-laserfiche-lite/sass/_dropdown-lf.scss:3]: what is the difference between those two?

File: projects/styles/lf-ms-office/sass/_dropdown-mo.scss [hunk 2]
```diff
@@ -21,7 +21,7 @@ mat-label.lf-dropdown-label {
   height: $height;
 }
 
-.mat-form-field-wrapper .mat-form-field-label-wrapper  .mat-form-field-label {
+.lf-label .mat-form-field-wrapper .mat-form-field-label-wrapper  .mat-form-field-label {
```
Comment (Ke-Jin-Laserfiche) [inline projects/styles/lf-ms-office/sass/_dropdown-mo.scss]: do we have a class called lf-label?

File: projects/styles/lf-laserfiche-lite/sass/rwc-checklist-lf.scss [hunk 3]
```diff
@@ -17,7 +17,7 @@
   font: 400 14px "Open Sans";
 }
 
-.mat-form-field-appearance-outline.mat-focused .mat-form-field-outline-thick {
+.lf-checklist .mat-form-field-appearance-outline.mat-focused .mat-form-field-outline-thick {
```
Comment (Ke-Jin-Laserfiche) [inline projects/styles/lf-laserfiche-lite/sass/rwc-checklist-lf.scss]: is lf-checklist a class name or a element tag name?

File: projects/styles/lf-laserfiche-lite/sass/_dropdown-lf.scss [hunk 4]
```diff
@@ -84,6 +86,6 @@ div.mat-select-value {
   border-top: 0 solid transparent;
 }
 
-.lf-dropdown.mat-form-field-appearance-legacy .mat-form-field-infix {
+.lf-dropdown.mat-form-field.lf-field > .mat-form-field-wrapper > .mat-form-field-flex > .mat-form-field-infix {
```
Comment (Ke-Jin-Laserfiche) [inline projects/styles/lf-laserfiche-lite/sass/_dropdown-lf.scss:85]: please add 
```
.lf-dropdown.mat-form-field-appearance-legacy .mat-form-field-label {
  top: 0;
}
```

File: projects/styles/lf-ms-office/sass/_dropdown-mo.scss [hunk 5]
```diff
@@ -87,6 +86,6 @@ div.mat-select-value {
   border-top: 0 solid transparent;
 }
 
-.lf-dropdown.mat-form-field-appearance-legacy .mat-form-field-infix {
+.lf-dropdown.mat-form-field.lf-field > .mat-form-field-wrapper > .mat-form-field-flex > .mat-form-field-infix {
```
Comment (Ke-Jin-Laserfiche) [inline projects/styles/lf-ms-office/sass/_dropdown-mo.scss:85]: please add
```
.lf-dropdown.mat-form-field-appearance-legacy .mat-form-field-label {
  top: 0;
}
```

File: projects/styles/lf-ms-office/sass/rwc-checklist-mo.scss [hunk 6]
```diff
@@ -17,7 +17,7 @@
   font: 400 14px "Segoe UI", "Segoe UI Web (West European)", "Segoe UI", -apple-system, BlinkMacSystemFont, Roboto, "Helvetica Neue", sans-serif;
 }
 
-.mat-form-field-appearance-outline.mat-focused .mat-form-field-outline-thick {
+.lf-checklist .mat-form-field-appearance-outline.mat-focused .mat-form-field-outline-thick {
```
Comment (Ke-Jin-Laserfiche) [inline projects/styles/lf-ms-office/sass/rwc-checklist-mo.scss]: the same as above. is this lf-checklist without the dot?

File: projects/styles/lf-laserfiche-lite/sass/_dropdown-lf.scss (hunk 1 above)
Comment (alexgomezlf) [reply projects/styles/lf-laserfiche-lite/sass/_dropdown-lf.scss:3]: there isn't one, I used a formatter on the file and it changed them to lower case

File: projects/styles/lf-ms-office/sass/_dropdown-mo.scss (hunk 2 above)
Comment (alexgomezlf) [reply projects/styles/lf-ms-office/sass/_dropdown-mo.scss]: ahh actually this may have been what broke the other styles.. I thought I added this class

**Review by Ke-Jin-Laserfiche — APPROVED** [review]

**Comment by Ke-Jin-Laserfiche** [comment]
fix #41

**Pull request merged**